docs/changelog/110993.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 110993
+summary: (Err>Doc) Max Shards Per Node
```

Comment:
This summary will appear in the release notes - could you edit it (and ideally the PR title) to make it more release-notes-worthy?